website/addons/base/views.py

```diff
@@ -202,7 +204,27 @@ def create_waterbutler_log(payload, **kwargs):
     auth = Auth(user=user)
     node = kwargs['node'] or kwargs['project']
 
-    if action in (NodeLog.FILE_MOVED, NodeLog.FILE_COPIED):
+    try:
+        dest = payload['destination']
```

Comment:
You can use `dict.get` instead of indexing here; this should obviate the need to handle `KeyError`.